spec/requests/dashboard_spec.rb

```diff
@@ -2,27 +2,54 @@
 
 require 'rails_helper'
 
-RSpec.describe '/dashboard' do
-  let(:user) { create(:user, organizations: [organization]) }
+RSpec.describe '/{organization_id}/dashboard', type: :request do
   let(:organization) { create(:organization) }
+  let(:user) { create(:user, organizations: [organization]) }
+
+  it_behaves_like 'protected' do
+    before { get organization_dashboard_path(organization, as: create(:user)) }
+  end
+
+  it_behaves_like 'unauthenticated' do
+    before { get organization_dashboard_path(organization) }
+  end
+
+  context 'with a user of the organization' do
+    subject { -> { get organization_dashboard_path(organization, as: user) } }
+
+    it 'renders the dashboard for the organizations user' do
+      subject.call
+      expect(response).to be_successful
+    end
+
+    it 'counts the active contributors for the organization' do
+      create_list(:contributor, 3, organization: organization)
+      create(:contributor)
+      subject.call
+      expect(page).to have_content '3 aktive Mitglieder'
+    end
+
+    it 'counts the requests for the organization' do
+      create_list(:request, 3, organization: organization)
+      create(:request)
+      subject.call
+      expect(page).to have_content '3 Fragen gestellt'
+    end
+
+    it 'counts the replies for the organization' do
+      create_list(:message, 3, organization: organization)
+      create(:message) # other message
```

Comment:
```suggestion
      create(:message, :inbound) # other message
      create(:message, :outbound) # shouldn't be counted
```
This is inconsistent with lines 50-51... By default, the message, from FactoryBot, is created to be inbound/ a reply so it can be omitted here and it be counted by the engagement metric. On lines 50-51, you did not omit the inbound trait and favored being more explicit.

I would favor being explicit here. I am the one that added that the message is inbound by default and I still needed to look at the factories definition to make sure what was happening here.

Reply:
yes thnx, being as self explanatory as possible is good.  Then we should also change line 40 to be the same. Just followed what has been there...